tests/test_init.py

```diff
@@ -489,6 +489,32 @@ async def connect(self):
                         body=[
                             {
                                 "/other": {},
+                                "/org/bluez/hci0/dev_54_D2_72_AB_35_95": {
```

Comment:
I think we should keep both tests. Ie copy the original one and add a new one like this so we can be sure it works both ways and we don't break it in the future

Reply:
Done.